packages/fxa-settings/src/components/PageRecoveryKeyAdd/index.tsx

```diff
@@ -1,20 +1,20 @@
-import React, { useCallback, useEffect, useState } from 'react';
 import { Localized, useLocalization } from '@fluent/react';
+import { RouteComponentProps, useNavigate } from '@reach/router';
 import base32encode from 'base32-encode';
+import React, { useCallback, useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
-import { RouteComponentProps, useNavigate } from '@reach/router';
-import { useAccount, useAlertBar } from '../../models';
-import InputPassword from '../InputPassword';
-import FlowContainer from '../FlowContainer';
-import VerifiedSessionGuard from '../VerifiedSessionGuard';
-import DataBlock from '../DataBlock';
 import { HomePath } from '../../constants';
-import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
 import {
-  AuthUiErrors,
   AuthUiErrorNos,
+  AuthUiErrors,
   composeAuthUiErrorTranslationId,
 } from '../../lib/auth-errors/auth-errors';
+import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
+import { useAccount, useAlertBar } from '../../models';
+import DataBlock from '../DataBlock';
+import FlowContainer from '../FlowContainer';
+import InputPassword from '../InputPassword';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
```

Comment:
This diff makes me really wonder about my Prettier settings, sigh, might see about reverting some of these import changes but I also want us to move over to [this anyway](https://www.npmjs.com/package/prettier-plugin-organize-imports)